onnxscript/_internal/ast_utils.py

```diff
@@ -3,9 +3,12 @@
 
 import ast
 import inspect
+import sys
 import textwrap
 import types
 
+py_version_ge_39 = sys.version_info[:2] >= (3, 9)
```

Comment:
```suggestion
PY_VERSION_GE_39 = sys.version_info >= (3, 9)
```

Upper case, also this seems to work: https://github.com/asottile/flake8-2020#rationale

Reply:
Also move this to version_utils?